v20.04/overlay/etc/templates/config.php

```diff
@@ -355,6 +355,10 @@ function getConfigFromEnv() {
     $config['excluded_directories'] = explode(',', getenv('OWNCLOUD_EXCLUDED_DIRECTORIES'));
   }
 
+  if (getenv('OWNCLOUD_INTEGRITY_CHECK_DISABLED') != '') {
+    $config['integrity.check.disabled'] = getenv('OWNCLOUD_INTEGRITY_CHECK_DISABLED') == 'false';
```

Comment:
```suggestion
    $config['integrity.check.disabled'] = getenv('OWNCLOUD_INTEGRITY_CHECK_DISABLED') == 'true';
```

The negation of a disable flag ... is ... naaah....

Reply:
no no, the default should be `false`

Reply:
In english: the default case should have the integrity check enabled